### PR TITLE
Add a paragraph on the idp service to the LAPI documentation

### PR DESCRIPTION
**Pull request by mziolekda**

### CODE REVIEW

File: docs/2.7.0/docs/app-dev/services.rst [hunk 1]
```diff
@@ -263,6 +263,18 @@ Read the :doc:`Authorization documentation </app-dev/authorization>` to understa
 
 User management is available in Canton-enabled drivers and not yet available in the Daml for VMware Blockchain driver.
 
+
+.. _identity-provider-config-service:
+
+Identity Provider Config Service
+================================
+
+Use **identity provider config service** to define and manage the parameters of an external IDP systems configured to issue tokens for a participant node.
```
Comment (carrielaben-da) [inline docs/2.7.0/docs/app-dev/services.rst:272]:
```suggestion
Use the **identity provider config service** to define and manage the parameters of the external IDP system configured to issue tokens for a participant node.
```

File: docs/2.7.0/docs/app-dev/services.rst [hunk 2]
```diff
@@ -263,6 +263,18 @@ Read the :doc:`Authorization documentation </app-dev/authorization>` to understa
...
+
+The **identity provider config service** makes it possible for participant node administrators to set up and manage additional identity providers at runtime. This allows using access tokens from identity providers unknown at deployment time. When an identity provider is configured, independent IDP administrators can manage their own set of parties and users.
```
Comment (carrielaben-da) [inline docs/2.7.0/docs/app-dev/services.rst:274]:
```suggestion
The **identity provider config service** allows participant node administrators to set up and manage additional identity providers at runtime. This means that the node can accept access tokens from identity providers unknown at deployment time. When an identity provider is configured, independent IDP administrators can manage their own set of parties and users.
```

**Review by carrielaben-da — APPROVED** [review]
A couple of small adjustments.

File: docs/2.7.0/docs/app-dev/services.rst [hunk 3]
```diff
@@ -263,6 +263,18 @@ Read the :doc:`Authorization documentation </app-dev/authorization>` to understa
...
+
+Such parties and users have a matching identity_provider_id defined and are inaccessible to administrators from other identity providers. A user will only be authenticated if the corresponding JWT token is issued by the appropriate identity provider. Users and parties without identity_provider_id defined are assumed to be using the default identity provider, which is configured statically when the participant node is deployed..
```
Comment (miklos-da) [inline docs/2.7.0/docs/app-dev/services.rst:276]: Nit: please remove the double dots at the end of the sentence.

**Pull request merged**